src/edgepi/adc/edgepi_adc.py

```diff
@@ -641,12 +640,16 @@ def reset(self):
         self.transfer(self.adc_ops.reset_adc())
         self.__reapply_config()
 
-    def __is_data_ready(self):
+    def __is_data_ready(self, adc_num: ADCNum):
         # pylint: disable=unused-private-member
         # required for integration testing in test_conversion_times.py
         """Utility for testing conversion times, returns True if ADC indicates new voltage data"""
-        read_data = self.transfer([ADCComs.COM_RDATA1.value] + [255] * 6)
-        return read_data[1] & 0b01000000
+        read_data = self.transfer([adc_num.value.read_cmd] + [255] * 6)
+        if adc_num is ADCNum.ADC_1:
+            ready = (read_data[1] & 0b01000000) == 0b01000000
+        if adc_num is ADCNum.ADC_2:
+            ready = (read_data[1] & 0b10000000) == 0b10000000
+        return ready
```

Comment:
is there a reason you are using bits instead of shorthand like hex?

Reply:
to be explicit which bit is responsible for ADC2 and ADC1 :)